db/migrate/20200508195858_add_avatar_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddAvatarToUsers < ActiveRecord::Migration[5.1]
+  def change
+    add_reference :users, :avatar, index: true
```

Comment:
I'm not sure an index is necessary here as I don't think we'll filter users based on their avatar, right?

Reply:
I indexed it as most (if not all) foreign keys were indexed too, but your observation is right. I'll rewrite this migration so it's cleaner.